translations/ui-organizations/en.json

```diff
@@ -315,6 +315,9 @@
   "summary.code": "Code",
   "summary.description": "Description",
   "summary.organizationStatus": "Organization status",
+  "summary.organizationStatus.active": "Active",
```

Comment:
I think it should be just `"organizationStatus.active": "Active",`

Reply:
Updated